dags/multipod/maxtext_configs_aot_hybridsim.py

```diff
@@ -72,7 +80,7 @@
           # Run AOT workload: generate HLO, upload to GCS
           aot_cmd = (
               'export XLA_FLAGS="--xla_dump_to=/tmp/xla_dump/"',
-              f"bash MaxText/configs/v{tpu.value}/{model_size}.sh EXECUTABLE=train_compile.py M_COMPILE_TOPOLOGY=v{tpu.value}-{num_cores} M_COMPILE_TOPOLOGY_NUM_SLICES={n}",
+              f"bash MaxText/configs/v{v5e_alt if tpu.value == TpuVersion.V5E.value else tpu.value}/{model_size}.sh EXECUTABLE=train_compile.py M_COMPILE_TOPOLOGY=v{v5e_alt if tpu.value == TpuVersion.V5E.value else tpu.value}-{num_cores} M_COMPILE_TOPOLOGY_NUM_SLICES={n}",
```

Comment:
Is this coming from `v5litepod` vs `v5e`? 😢

Reply:
Yes unfortunately 😢